### PR TITLE
trim how we are using same help text over and over again

### PR DESCRIPTION
**Pull request by gauravsaralMs**
Please make sure the code is following contribution guidelines in CONTRIBUTING.md

 - [ ] : This PR has a corresponding issue open in the Repository.
 - [ ] : Approach is signed off on the issue.

### CODE REVIEW

File: azure-devops/azext_devops/dev/team/arguments.py [hunk 1]
```diff
@@ -22,27 +21,24 @@
                                           SERVICE_ENDPOINT_AUTHORIZATION_SERVICE_PRINCIPAL]
 
 
+# pylint: disable=line-too-long
```
Comment (atbagga) [inline azure-devops/azext_devops/dev/team/arguments.py]: This one can be fixed. Line length limit is to improve readability. 
We should not disable all pylint warnings to fix CI :)

File: azure-devops/azext_devops/dev/repos/pull_request.py [hunk 2]
```diff
@@ -82,10 +78,6 @@ def list_pull_requests(repository=None, creator=None, include_links=False, revie
     :type skip: int
     :param top: Maximum number of pull requests to list.
     :type top: int
```
Comment (atbagga) [inline azure-devops/azext_devops/dev/repos/pull_request.py]: Project help text instance here in list_pull_requests command. 

Probably a one off miss but difficult to find in the diff if any more are missed. Just double check once before merging.

**Comment by atbagga** [comment]
There are couple of comments but approving since the changes do not need a review again.

File: azure-devops/azext_devops/dev/team/arguments.py (hunk 1 above)
Comment (gauravsaralMs) [reply azure-devops/azext_devops/dev/team/arguments.py]: agree, but it's hard to validate style changes locally right now

File: azure-devops/azext_devops/dev/repos/pull_request.py (hunk 2 above)
Comment (gauravsaralMs) [reply azure-devops/azext_devops/dev/repos/pull_request.py]: agree, lets clean/fix it as and when we find anything

**Pull request merged**